docs/guides/nrfconnect_examples_cli.md

```diff
@@ -0,0 +1,74 @@
+# Using CLI in nRF Connect SDK examples
+
+The following examples for the development kits from Nordic Semiconductor include a command-line interface
+that allows access to application logs and [Zephyr shell](https://docs.zephyrproject.org/1.13.0/subsystems/shell.html):
+
+* [Lock app example](../../examples/lock-app/nrfconnect/README.md)
+* [Lighting app example](../../examples/lighting-app/nrfconnect/README.md)  
+
+## Accessing the CLI console
+
+To access the CLI console, use a serial terminal emulator of your choice.
+For example, you can use Minicom or GNU Screen.
+Use the baud rate set to `115200`.
+
+For example, to start using the CLI console with Minicom, complete the following steps:
+1. Run the following command with */dev/ttyACM0* replaced with a device node of your development kit.
+
+    $ minicom -D /dev/ttyACM0 -b 115200
+
+2. Reboot the board to see the boot logs in the console:
```

Comment:
@greg-fer Rebooting isn't necessary to use the CLI, it was only mentioned as it triggers many logs, so one can easily see if the console works correctly. Perhaps we shouldn't mention it as one of the points needed to "start using the CLI console"?

Reply:
Updated.

Reply:
Thanks, everything else is looking good!